tmail-backend/apps/distributed/pom.xml

```diff
@@ -83,6 +83,11 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>tmail-rate-limiter-cassandra</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>tmail-webadmin-mailbox</artifactId>
+            <version>${project.version}</version>
```

Comment:
Actually you should declare `tmail-webadmin-mailbox` in the parent pom (`tmail-backend/pom.xml`) like for the other modules, to lock down its declaration in just one place. Then you can refer refer it in other modules (like this one) without repeating the version